src/test/resources/schemas/medium.json

```diff
@@ -47,6 +47,10 @@
           "Sonstige"
         ]
       }
```

Comment:
Wasn't there a way to associate corresponding URIs and strings with one another in the JSON Schema? However, it probably isn't important given how we do the transformation.